src/debug/fee-update-operations.json

```diff
@@ -0,0 +1,6 @@
+{
+  "feeUpdateOperations": [
+    { "walletId": "11d2a204-9981-420f-a148-020b7d52607f", "fee": 13 },
```

Comment:
These aren't real walletIds right?

Reply:
They're valid but are from local env

Reply:
I guess they're not valid then, because they're been created randomly

Reply:
The term "valid" would imply that the value has passed the validation checks, which is true for this value.
The validation check being a regex.